contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -40,26 +41,23 @@ contract MixinStorage is
     // address of staking contract
     address internal stakingContract;
 
-    // mapping from Owner to Amount Staked
-    mapping (address => uint256) internal stakeByOwner;
+    // mapping from Owner to Amount of Active Stake
+    mapping (address => IStructs.DelayedBalance) internal activeStakeByOwner;
```

Comment:
Nit: all of these variables should be prefixed with `_`. Probably cleaner to do this in another PR.

Reply:
Agreed, I've created a task for this.